external/src/test/groovy/org/hawkular/alerts/rest/AbstractQuarkusITestBase.groovy

```diff
@@ -14,7 +14,7 @@ import java.util.concurrent.atomic.AtomicInteger
  */
 abstract class AbstractQuarkusITestBase {
 
-    static baseURI = 'http://127.0.0.1:8084/hawkular/alerts/'
+    static baseURI = 'http://127.0.0.1:8095/hawkular/alerts/'
```

Comment:
We should (not necessarily as part of this PR) let the port be injected by the Quarkus test framework (see e.g. `@QuarkusTest`)